modules/appeals_api/spec/spec_helper.rb

```diff
@@ -22,4 +22,8 @@
   Settings.modules_appeals_api.documentation.wip_docs&.each do |flag|
     config.filter_run_excluding "wip_doc_#{flag}".to_sym unless wip_docs_enabled.include?(flag)
   end
+
+  # RSwag specs won't run correctly unless certain environment variables are set correctly via rake tasks -
+  # they should be skipped in the course of normal rspec runs:
+  config.filter_run_excluding(rswag_only: true) if ENV['SWAGGER_DRY_RUN'].blank?
```

Comment:
When we run rswag specs, certain environment variables are required to be set, or the specs will fail to set up. These required environment variables are set in our `rswag.rake` tasks, but the same specs may also be picked up as part of a normal `bundle exec rspec` run. In this latter context, we used to be able to default to using values from the decision reviews API v2 when the environment variables were unset. But now that we've added another set of contestable issues specs, it's much harder to resolve valid defaults in `rswag_config.rb` when these specs run under rspec. To avoid this problem, I set this up so that including the `rswag_only: true` tag in an rswag example will skip them when running anywhere besides via rake tasks

Reply:
I would double check on this - I believe we will want these specs to run in every context, as rswag is itself a spec runner that validates that our docs match our output. 

I feel like there are likely cases where we've relied on rswag to let us know, for example, that a certain response code is being returned as expected that our controller specs aren't, and therefore skipping rswag runs in a regular `bundle exec rspec` would be a false green run if we started returning improper codes.

Reply:
Ok thanks - I think I've managed this - I just needed to make a few updates to the DocHelpers

Reply:
Oops, realized I had forgotten to actually remove the addition to the spec_helper itself - all done now.